src/Control/Monad/Logic/Sequence.hs

```diff
@@ -22,6 +22,7 @@ module Control.Monad.Logic.Sequence
   , getSeq
 #endif
   , View(..)
+  , alt
```

Comment:
I'm not sure if I actually want to export `alt`, `asumSeqT`, and `chooseSeqT`. I think I mainly added them to test things when rewrites were not working for me.